src/transformers/integrations/peft.py

```diff
@@ -132,6 +133,8 @@ def load_adapter(
         check_peft_version(min_version=MIN_PEFT_VERSION)
 
         adapter_name = adapter_name if adapter_name is not None else "default"
+        if adapter_hub_kwargs is None:
+            adapter_hub_kwargs = {}
```

Comment:
Why not default to {} directly ?

Reply:
Better this way! https://www.inspiredpython.com/article/watch-out-for-mutable-defaults-in-function-arguments